src/datastore/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[package]
+name        = "google-cloud-datastore"
```

Comment:
do we plan to implement a veneer for this? is it sufficient to do `firestore`?

(I am fine to remove this later. I see that we are not publishing it.)

Reply:
I am not sure. I would prefer to just generate Firestore. Datastore usage is still high so we may want to generate both.